text/0003-measure-metric-type.md

```diff
@@ -124,9 +124,17 @@ The optional properties of a metric instrument are:
 |----------|-------------|-------------|
 | Required Keys | Determines labels that are always set on metric handles | All kinds |
 | Disabled | Indicates a verbose metric that does not report by default | All kinds |
-| Bidirectional | Indicates a cumulative metric instrument that goes up and down | Cumulative |
-| Unidirectional | Indicate a gauge that only ascends, for rate calculation | Gauge |
-| NonNegative | Indicates a measure that is never negative, for rate calculation | Measure |
+| NonMonotonic | Indicates a cumulative metric instrument that goes up and down | Cumulative |
+| Monotonic | Indicate a gauge that expresses a monotonic cumulative value | Gauge |
+| Signed | Indicates a measure that is positive or negative | Measure |
```

Comment:
Can we remove the signed as we discussed in one of the meeting?

Reply:
It's a meaningful distinction to keep IMO.
Examples were provided in the referenced thread, e.g., for recording spanAsssasin scores. For recording the change in a bank account, there are just so many examples. 

Why should the implementation care? What value is there to the implementation in not having to support negative values? I can think of one. For computing summary metrics, i.e., quantile estimates, we need an approximation algorithm. It is significantly easier to approximate a "one-sided" distribution that has one boundary.  It's significantly easier to approximate a "two-sided" distribution that has no boundaries. If you're told the measure is non-negative you might choose to use a [DDSketch](https://arxiv.org/pdf/1908.10693.pdf).  If you're told the measure is signed, you might choose to use a [T-digest ](https://github.com/tdunning/t-digest/blob/master/docs/t-digest-paper/histo.pdf). 

I don't see this as being expensive to keep in, but I see it as a lack of symmetry if we exclude it.  We shouldn't be so confident that negative measures are not useful.